style/reset.css

```diff
@@ -59,11 +59,22 @@ html {
   .product-list-container {
     grid-template-areas:
       "header header header header header header"
-      "product product product product product product";
+      "menu menu product product product product";
+      
   }
   .header-main-content {
     display: none;
   }
+}
+
+@media only screen and (max-width: 500px) {
+  .product-list-container {
+    grid-template-areas:
+      "header"
+      "filter-mobile-layout"
```

Comment:
I think filter-mobile-layout is menu as you have used before, why did you change it? Just curious to know

Reply:
> I think filter-mobile-layout is menu as you have used before, why did you change it? Just curious to know

Thanks for the review. I have changed it as I wanted to modify its layout.